enhancements/installer/openstack-supported-versions.md

```diff
@@ -0,0 +1,218 @@
+---
+title: openstack-bring-your-own-external-connectivity
+authors:
+  - "@EmilienM"
+  - "@mdbooth"
+reviewers:
+  - "@mandre"
+  - "@pierreprinetti"
+approvers:
+  - "@mandre"
+  - "@pierreprinetti"
+creation-date: 2020-11-04
+last-updated: 2020-11-04
+status: implementable
+---
+
+# OpenStack supported versions
+
+- [OpenStack supported versions](#openstack-supported-versions)
+  - [Release Signoff Checklist](#release-signoff-checklist)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+  - [Proposal](#proposal)
+    - [Design Details](#design-details)
+    - [User Stories](#user-stories)
+      - [Check the OpenStack version](#check-the-openstack-version)
+      - [Ignore the version check](#ignore-the-version-check)
+        - [Example Usage](#example-usage)
+    - [Test Plan](#test-plan)
+    - [Graduation Criteria](#graduation-criteria)
+      - [Dev Preview -> Tech Preview](#dev-preview---tech-preview)
+      - [Tech Preview -> GA](#tech-preview---ga)
+    - [Infrastructure Needs](#infrastructure-needs)
+    - [Drawbacks](#drawbacks)
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+OpenShift can only be installed on supported versions of OpenStack.
+While the [Support Matrix](https://access.redhat.com/articles/4679401) is
+well documented, the OpenShift installer doesn't check which version of
+OpenStack is running before attempting an operation (e.g. installation). 
+Running OpenShift over an unsupported version of OpenStack is problematic
+and can lead to unstable infrastructure. To avoid that, it is proposed
+to programmatically verify the version of OpenStack is supported
+otherwise stop and fail.
+
+## Motivation
+
+### Goals
+
+- Disallow the installation of OpenShift done via the installer to run
+  against a cloud where the OpenStack version is not supported.
+- Allow an administrator to ignore the OpenStack version check and force the
+  installer to proceed anyway. This can be useful if the OpenStack cloud
+  is partially upgraded to a major version or if there is a Support
+  Exception granted to run that deprecated version of OpenStack.
+
+### Non-Goals
+
+- This enhancement does not intend do block any OpenShift installation done
```

Comment:
I don't think this captures the spirit of the initial bug/request and needs to be removed as a non-goal. IE: it needs to be a goal or target.

Reply:
I removed it.